pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -66,10 +66,14 @@
   public static final String CONTROLLER_MODE = "controller.mode";
   public static final String LEAD_CONTROLLER_RESOURCE_REBALANCE_STRATEGY = "controller.resource.rebalance.strategy";
 
-  // Comma separated list of list of packages that contain TableConfigTuners to be added to the registry
+  // Comma separated list of packages that contain TableConfigTuners to be added to the registry
   public static final String TABLE_CONFIG_TUNER_PACKAGES = "controller.table.config.tuner.packages";
   public static final String DEFAULT_TABLE_CONFIG_TUNER_PACKAGES = "org.apache.pinot";
 
+  // Comma separated list of packages that contains javax service resources.
+  private static final String CONTROLLER_RESOURCE_PACKAGES = "org.apache.pinot.controller.api.resources";
```

Comment:
Should the package name contain a `.package` suffix here?

Reply:
Good point, how about rename it as:
```suggestion
  private static final String CONTROLLER_RESOURCE_PACKAGES = "controller.restlet.api.resource.packages";
```

Reply:
Looks good.